src/ifaddrs.rs

```diff
@@ -144,4 +176,21 @@ mod tests {
     fn test_getifaddrs() {
         let _ = getifaddrs();
     }
+
+    #[test]
```

Comment:
To a casual reader, this test looks pretty pointless.  Please add a comment explaining the rationale.